sources/packages/web/src/store/modules/institution/institution.ts

```diff
@@ -0,0 +1,27 @@
+import { Module } from "vuex";
+
+import {
+  InstitutionLocationState,
+  RootState,
+  InstitutionUserAndAuthDetailsForStore,
+  InstitutionLocationsDetailsForStore,
+} from "@/types";
+
+import { actions } from "./actions";
+import { getters } from "./getters";
+import { mutations } from "./mutations";
+
+export const state: InstitutionLocationState = {
+  myInstitutionAndUserDetailsState: {} as InstitutionUserAndAuthDetailsForStore,
```

Comment:
Just a recommendation...
These 2 variables (myInstitutionAndUserDetailsState and myInstitutionLocationsState) seems to be too verbose, IMO.
If you agree, myInstitutionLocationsState could be renamed to something like "locationState" since it will be accessed like below.

```
store.state.institution.locationState
```

myInstitutionAndUserDetailsState could be renamed and have their inner objects moved as below.

```
store.state.institution.userState
store.state.institution.authorizationsState
```